### PR TITLE
Mention the need for zones with NVIDIA Tesla K80 Accelerators for the default GCP configuration

### PR DESCRIPTION
**Pull request by yashjakhotiya**
Solves #1987

### CODE REVIEW

**Comment by janeman98** [comment]
@abhi-g 
Can you (or find someone in your company to) review this PR? Thanks!

**Comment by joeliedtke** [comment]
preview: https://deploy-preview-1988--competent-brattain-de2d6d.netlify.app/docs/gke/deploy/deploy-cli/

**Comment by joeliedtke** [comment]
@Bobgy, do you know if the following manifest requires NVIDIA K80 GPUs?

https://raw.githubusercontent.com/kubeflow/manifests/v1.0-branch/kfdef/kfctl_gcp_iap.v1.0.2.yaml

**Comment by Bobgy** [comment]
Yes, that's true.
Thanks for updating the doc!
/lgtm
/approve

**Comment by Bobgy** [comment]
@yashjakhotiya Can you please resolve the merge conflict?
Thanks!

**Comment by 8bitmp3** [comment]
Hi @yashjakhotiya I hope all is good! Please resolve the merge issue (cc @Bobgy @joeliedtke @RFMVasconcelos). This PR has gone a bit stale

**Comment by Bobgy** [comment]
I've helped resolve merge conflicts and update some content.
/assign @8bitmp3 
do you want to review?

File: content/en/docs/gke/deploy/deploy-cli.md [hunk 1]
```diff
@@ -121,6 +121,20 @@ Refer to
...
+  For example,  `my-kubeflow` or `kf-test`.
+  The value of KF_NAME must consist of lower case alphanumeric characters or
+  '-', and must start and end with an alphanumeric character.
+  The value of this variable cannot be greater than 25 characters. It must
+  contain just a name, not a directory path.
+  You also use this value as directory name when creating the directory where 
+  your Kubeflow  configurations are stored, that is, the Kubeflow application 
+  directory. 
+  
+* **${KF_DIR}** - The full path to your Kubeflow application directory.
```
Comment (8bitmp3) [inline content/en/docs/gke/deploy/deploy-cli.md:135]:
```suggestion
* **${KF_DIR}**: The full path to your Kubeflow application directory.
```

**Review by 8bitmp3 — CHANGES_REQUESTED** [review]
Hey @yashjakhotiya Hope you didn't have to wait for too long, I've added a few suggestions. They're mainly about formatting code blocks.

Can you please lint the entire `deploy-cli.md` and add `shell` next to three backticks in the beginning of code blocks to improve formatting in Markdown on the site?

For example:

```
    ```shell
    gcloud ....
    ```
```

Thank you


cc @Bobgy

**Comment by Bobgy** [comment]
@8bitmp3 I don't think @yashjakhotiya has responded since creating the PR.
Let's get this in, because it's already an improvement
/lgtm
/approve

**Pull request merged**